helm-chart/metabase/values.yaml

```diff
@@ -0,0 +1,191 @@
+replicaCount: 1
+
+image:
+  repository: metabase/metabase
+  pullPolicy: IfNotPresent
+  command: []
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  create: true
+  annotations: {}
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  name: metabase
+  type: ClusterIP
+  externalPort: 80
+  internalPort: 3000
+
+livenessProbe:
+  initialDelaySeconds: 120
+  timeoutSeconds: 30
+  failureThreshold: 6
+
+readinessProbe:
+  initialDelaySeconds: 30
+  timeoutSeconds: 3
+  periodSeconds: 5
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+  hosts:
+    - host: metabase.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+
+config:
```

Comment:
This (listing every env var) will be a maintenance burden.
Also, it would be good to have an `extraArgs: {}` and an `extraEnvs: []` field so the chart can still be used even if it becomes outdated somehow.

Reply:
We shouldn't be removing Env vars, only adding. Just added extraArgs and extraEnvs, please take a look

Reply:
Done